src/index.ts

```diff
@@ -306,52 +322,70 @@ function convertLegacyMsg(msg: amino.AminoMsg): Pick<ProtoTx, 'msg'> {
       }
     case 'osmosis/gamm/swap-exact-amount-in':
       if (!msg.value.sender) throw new Error('Missing sender in msg')
-      if (!msg.value.tokenIn) throw new Error('Missing tokenIn in msg')
-      if (!msg.value.tokenOutMinAmount) throw new Error('Missing tokenOutMinAmount in msg')
+      if (!msg.value.token_in) throw new Error('Missing token_in in msg')
```

Comment:
seems like we could support both camel and snake case value keys if we wanted to

Reply:
We could support both if we wanted to, but I don't think we do. Unifying the case style across the stack is the way to go, IMO - having different tx formats in different places is a PITA and expands the surface for potential bugs.